packages/block-library/src/site-logo/editor.scss

```diff
@@ -93,3 +93,65 @@
 		}
 	}
 }
+
+.block-library-site-logo__inspector-upload-container {
+	position: relative;
+	// Since there is no option to skip rendering the drag'n'drop icon in drop
+	// zone, we hide it for now.
+	.components-drop-zone__content-icon {
+		display: none;
+	}
+}
+
+.block-library-site-logo__inspector-upload-container,
+.block-library-site-logo__inspector-media-replace-container {
+	button.components-button {
+		color: $gray-900;
```

Comment:
most of these styles were copied from the `style variations` buttons. Should we consider a new 'variant' of button if we plan on reusing them? 🤔

Reply:
Reusing is my only remaining concern about this PR, because exatcly yes, we should have as little custom CSS as possible in PRs like these, and ideally reuse componentry whenever possible.

What would it require in this place? Do the style variation buttons need to be canonized as part of the Button style? Or is it a toggle? And if we canonize those can we reuse them here? If either answer is _yes_, it would be great to do that, either here or as a followup. Thanks so much for working on this.

Reply:
In my last commit I removed some obsolete css, so it's way less now. Aside of that, these outline buttons seem to be a combination of `secondary` variant Buttons and a few overrides. I think it would make sense to have a specific Button variant for such styles.

Noting though that these styles can be used probably in just a `div` or something and it would be good to have a common styling - not sure where it should live in code though.. An example is this PR too, where we use similar styles(not the hover ones) for users that have no permissions to update, so we render the info in a `div`.

I think we can handle this in a follow up, but 🤷 . Whatever you prefer.